app/assets/stylesheets/application.css.scss

```diff
@@ -26,16 +26,22 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
 
 
 .post {
-  border-top: solid black 1px;
+  border-radius: 10px;
+  margin: 20px 10px;
+  padding: 5px 20px;
+  background-color: #fbfbfb;
```

Comment:
Color literals like `#fbfbfb` should only be used in variable declarations; they should be referred to via variable everywhere else.